app/assets/stylesheets/forever_style_guide/base/_grid.scss

```diff
@@ -0,0 +1,136 @@
+$f-grid-default-padding: 2px;
+$f-grid-bootstrap-padding: 15px;
+$f-grid-col-count: 24;
+
+@mixin f-grid-make-column($size, $columns: $f-grid-col-count) {
+  max-width: percentage($size / $columns);
+  flex: 0 0 percentage($size / $columns);
+}
+
+@mixin f-grid-make-column-offset($size, $columns: $f-grid-col-count) {
+  margin-left: percentage($size / $columns);
+}
+
+@mixin f-grid-make-columns-for-breakpoint($bp, $columns: $f-grid-col-count) {
+  @for $i from 1 through $f-grid-col-count {
+    .f-col-#{$bp}-#{$i} {
+      @include f-grid-make-column($i, $f-grid-col-count);
+    }
+
+    .f-col-#{$bp}-offset-#{$i} {
+      @include f-grid-make-column-offset($i, $f-grid-col-count)
+    }
+  }
+
+  //no flex inside column
+  .f-col-#{$bp}-none {
+    max-width: none;
+    flex: none;
+  }
+
+  //auto-width
+  .f-col-#{$bp} {
+    flex-grow: 1;
+    flex-basis: 0;
+  }
+
+}
+
+.f-grid {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  margin-left: -$f-grid-default-padding;
+  margin-right: -$f-grid-default-padding;
+}
+
+[class*="f-col"] {
+  flex-basis: 100%;
+  min-height: 1px;
+  align-items: stretch;
+  padding: $f-grid-default-padding;
+}
+
+/* RESPONSIVE */
+@include f-grid-make-columns-for-breakpoint('xxs');
```

Comment:
'XXS' doesn't currently exist as a global breakpoint and since the rest of the declarations use min-widths instead of max-widths, XS will only target phablets and tablets (>480) instead of phones.